### ISSUE TITLE
PR for structuring the repository as a Python module

### ISSUE THREAD

**Issue by mturilli** [radical-collaboration/facts#9]

**Comment by mturilli** [comment]
Unblocked by closing #13.

**Comment by ggg121** [comment]
I spoke with Bob this morning.  We don't need FACTS packaged up for the meeting next week.  It would be nice to have a simple package put together to share among the development team, if time permits.  In the long term, we should have this packaged up for third-party development, but that's not a current priority.

**Comment by ggg121** [comment]
@karahbit set up FACTS as a python module.  He is currently working on containerization.

**Comment by ggg121** [comment]
FACTS module and containerization complete.  Closing issue.
